instrumentation/reactor/reactor-netty/reactor-netty-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/reactornetty/v1_0/ReactorNettyNetClientAttributesGetter.java

```diff
@@ -38,4 +54,21 @@ public InetSocketAddress getAddress(
     }
     return null;
   }
+
+  private static URI parseUri(HttpClientConfig request) {
+    String baseUrl = request.baseUrl();
+    String uri = request.uri();
+
+    URI parsedUri;
+    try {
+      if (baseUrl != null && uri.startsWith("/")) {
+        parsedUri = new URI(baseUrl);
+      } else {
+        parsedUri = new URI(uri);
+      }
+    } catch (URISyntaxException ignored) {
+      return null;
+    }
+    return parsedUri;
+  }
```

Comment:
I'd prefer to parse out host/port manually to avoid URI creation, I'm happy to do that (and the grpc one too) in a follow-up if you create tracking issue

Reply:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/6568